vdb-bench-assembly/plugins/vdb-bench-dataservice/content/css/styles.less

```diff
@@ -706,3 +706,16 @@ button[id^='dropdownKebabRight_'] {
     position: relative;
     top: -4em;
 }
+
+.dsb-wizard-container .modal-header {
+    padding: 0px 15px;
```

Comment:
Why does the wizard require a modal-header? Is this a leftover of the wizard being modal if required (but we're not using that option)?